--- /etc/fwupd/uefi_capsule.conf	2022-02-22 22:58:16.338381621 +0100
+++ /etc/fwupd/uefi_capsule.conf	2022-02-21 10:46:09.129979189 +0100
@@ -5,7 +5,7 @@
 
 # the shim loader is required to chainload the fwupd EFI binary unless
 # the fwupd.efi file has been self-signed manually
-#DisableShimForSecureBoot=true
+DisableShimForSecureBoot=true
 
 # the EFI system partition (ESP) path used if UDisks is not available
 # or if this partition is not mounted at /boot/efi, /boot, or /efi
