--- /etc/pacman.conf	2021-07-09 18:37:11.257162082 +0200
+++ /etc/pacman.conf	2021-07-09 18:37:01.567067769 +0200
@@ -22,7 +22,7 @@
 Architecture = auto
 
 # Pacman won't upgrade packages listed in IgnorePkg and members of IgnoreGroup
-#IgnorePkg   =
+IgnorePkg   = linux linux-headers linux-lts linux-lts-headers
 #IgnoreGroup =
 
 #NoUpgrade   =
@@ -30,11 +30,11 @@
 
 # Misc options
 #UseSyslog
-#Color
+Color
 #NoProgressBar
 CheckSpace
 #VerbosePkgLists
-#ParallelDownloads = 5
+ParallelDownloads = 10
 
 # By default, pacman accepts packages signed by keys that its local keyring
 # trusts (see pacman-key and its man page), as well as unsigned packages.
@@ -69,8 +69,8 @@
 # repo name header and Include lines. You can add preferred servers immediately
 # after the header, and they will be used before the default mirrors.
 
-#[testing]
-#Include = /etc/pacman.d/mirrorlist
+[testing]
+Include = /etc/pacman.d/mirrorlist
 
 [core]
 Include = /etc/pacman.d/mirrorlist
@@ -78,8 +78,8 @@
 [extra]
 Include = /etc/pacman.d/mirrorlist
 
-#[community-testing]
-#Include = /etc/pacman.d/mirrorlist
+[community-testing]
+Include = /etc/pacman.d/mirrorlist
 
 [community]
 Include = /etc/pacman.d/mirrorlist
@@ -87,14 +87,18 @@
 # If you want to run 32 bit applications on your x86_64 system,
 # enable the multilib repositories as required here.
 
-#[multilib-testing]
-#Include = /etc/pacman.d/mirrorlist
+[multilib-testing]
+Include = /etc/pacman.d/mirrorlist
 
-#[multilib]
-#Include = /etc/pacman.d/mirrorlist
+[multilib]
+Include = /etc/pacman.d/mirrorlist
 
 # An example of a custom package repository.  See the pacman manpage for
 # tips on creating your own repositories.
 #[custom]
 #SigLevel = Optional TrustAll
 #Server = file:///home/custompkgs
+
+[aur]
+SigLevel = Optional TrustAll
+Server = file:///var/cache/pacman/aur
