--- /etc/mkinitcpio.conf
+++ /etc/mkinitcpio.conf
@@ -49,7 +49,7 @@
 #
 ##   NOTE: If you have /usr on a separate partition, you MUST include the
 #    usr, fsck and shutdown hooks.
-HOOKS=(base udev autodetect modconf block filesystems keyboard fsck)
+HOOKS=(base systemd autodetect modconf block keyboard sd-encrypt filesystems)
 
 # COMPRESSION
 # Use this to compress the initramfs image. By default, zstd compression
