--- /etc/fwupd/uefi_capsule.conf	2022-01-04 23:33:58.158638824 +0100
+++ /etc/fwupd/uefi_capsule.conf	2021-08-07 00:19:51.324115971 +0200
@@ -5,7 +5,7 @@
 
 # the shim loader is required to chainload the fwupd EFI binary unless
 # the fwupd.efi file has been self-signed manually
-#DisableShimForSecureBoot=true
+DisableShimForSecureBoot=true
 
 # the EFI system partition path used
 # if this is is not /boot/efi, /boot, or /efi
